plugin-min/test__if_not_no-unit-tests/hello-world.spec.ext

```diff
@@ -1,38 +1,27 @@
-// @if mocha
-import { expect } from 'chai';
-// @endif
 // @if vitest
-import { describe, it, expect } from 'vitest';
-// @endif
-// @if shadow-dom
-import { CustomElement } from 'aurelia';
+import { describe, it } from 'vitest';
 // @endif
-import { render } from './helper';
 import { HelloWorld } from '../src/hello-world';
+import { createFixture } from '@aurelia/testing';
+// @if jest
+import { bootstrapTestEnvironment } from './setup';
 
-describe('hello-world', () => {
+beforeAll(bootstrapTestEnvironment);
+// @endif
+
+describe('my-app', () => {
   it('should render message', async () => {
-    const node = (await render('<hello-world message="foo"></hello-world>', HelloWorld)).firstElementChild;
-    // @if shadow-dom
-    // In Shadow DOM open mode, shadowRoot is also accessible through DOM API
-    //   node.shadowRoot
-    // But only Aurelia API can access shadowRoot in both open and closed mode.
-    const shadowRoot = CustomElement.for(node).shadowRoot;
-    // @if babel
-    const text = shadowRoot.textContent;
-    // @endif
-    // @if typescript
-    const text = (shadowRoot as Node).textContent;
-    // @endif
-    // @endif
-    // @if !shadow-dom
-    const text =  node.textContent;
-    // @endif
-    // @if jasmine || jest || vitest
-    expect(text.trim()).toBe('Hello World! foo');
-    // @endif
-    // @if mocha
-    expect(text.trim()).to.equal('Hello World! foo');
-    // @endif
+    const { assertText, startPromise, stop } = createFixture(
+      '<hello-world message="message"></hello-world>',
+      { message: "foo" },
+      [HelloWorld],
+    );
+
+    try {
```

Comment:
hmm interesting, this is still a bit too much ceremony. We have the hook `onFixtureCreated` from the testing package that can allow us to do something like this:

```ts
const fixtures = [];
beforeEach(() => {
  onFixtureCreated(fixture => {
    fixtures.push(fixture);
  });
});

afterEach(() => {
  fixtures.forEach(f => f.stop(true));
  fixtures.length = 0;
});
```
In a setup/preload file, if we do this, we won't need to do `try/catch` or manual stop in each test. And because `stop` is idempotent, so individual test can call stop to do their assertions without having to worry about these setup blocks.

Though having to do the above is also some ceremony in itself, maybe we should improve so that we can automatically connect to `beforeEach`/`afterEach`?

Reply:
I will try.

Reply:
Ready for review.